src/Widgets/Scale.vala

```diff
@@ -17,77 +17,59 @@
 
 public class Sound.Widgets.Scale : Gtk.Grid {
     private Gtk.Image image;
-    private Gtk.Switch switch_widget;
-    private Gtk.Scale scale_widget;
+
+    public bool active { get; set; }
+    public Gtk.Scale scale_widget { get; private set; }
 
     public Scale (string icon, bool active = false, double min, double max, double step) {
-        this.hexpand = true;
-        var image_box = new Gtk.EventBox ();
         image = new Gtk.Image.from_icon_name (icon, Gtk.IconSize.DIALOG);
-        image_box.halign = Gtk.Align.START;
-        image_box.add (image);
+        image.pixel_size = 48;
 
-        this.attach (image_box, 0, 0, 1, 1);
+        var image_box = new Gtk.EventBox ();
+        image_box.add (image);
 
         scale_widget = new Gtk.Scale.with_range (Gtk.Orientation.HORIZONTAL, min, max, step);
         scale_widget.margin_start = 6;
         scale_widget.set_size_request (175, -1);
         scale_widget.set_draw_value (false);
         scale_widget.hexpand = true;
-        this.attach (scale_widget, 1, 0, 1, 1);
 
-        switch_widget = new Gtk.Switch ();
+        var switch_widget = new Gtk.Switch ();
         switch_widget.active = active;
         switch_widget.valign = Gtk.Align.CENTER;
         switch_widget.margin_start = 6;
         switch_widget.margin_end = 12;
 
-        this.attach (switch_widget, 2, 0, 1, 1);
+        hexpand = true;
+        get_style_context ().add_class ("indicator-switch");
+        attach (image_box, 0, 0, 1, 1);
+        attach (scale_widget, 1, 0, 1, 1);
+        attach (switch_widget, 2, 0, 1, 1);
 
```

Comment:
@danrabbit just code style: `attach (image_box, 0, 0);` you could remove last two parameters. `1` is default value.

Reply:
Oh TIL. I just realized these are all horizontal so can just use `add` anyways :p